core/services/run_manager.go

```diff
@@ -178,6 +178,42 @@ func (rm *runManager) CreateErrored(
 	return &run, rm.orm.CreateJobRun(&run)
 }
 
+// If we have seen the same runRequest already, then double the required incoming confs.
+// This is a
+func (rm *runManager) MaybeDoubleMinIncomingConfs(run models.JobRun) (*models.JobRun, error) {
```

Comment:
hmm is there some specific reason why have non-pointer as a param (nil protection?), but still return a pointer to it from the function?  
As you are still modifying it deeper (run.TaskRuns[i].MinRequiredIncomingConfirmations.Uint32 = ... ) you could just have the pointer as  the param

Reply:
Kind of left over from an earlier approach - updated to just use the pointer